frontend/public/components/namespace.jsx

```diff
@@ -667,23 +668,28 @@ const ProjectRow = (rowArgs) => (
   />
 );
 
-export const ProjectsTable = (props) => (
-  <Table
-    {...props}
-    aria-label="Projects"
-    Header={projectHeaderWithoutActions}
-    Row={ProjectRow}
-    customData={{
-      ProjectLinkComponent: ProjectLink,
-      actionsEnabled: false,
-      isColumnManagementEnabled: false,
-    }}
-    virtualize
-  />
-);
+export const ProjectsTable = (props) => {
+  const { t } = useTranslation();
+  return (
+    <Table
+      {...props}
+      aria-label={t('namespace~Projects')}
+      Header={projectHeaderWithoutActions}
+      Row={ProjectRow}
+      customData={{
+        ProjectLinkComponent: ProjectLink,
+        actionsEnabled: false,
+        isColumnManagementEnabled: false,
+      }}
+      virtualize
+    />
+  );
+};
 
-const headerWithMetrics = () => projectTableHeader({ showMetrics: true, showActions: true });
-const headerNoMetrics = () => projectTableHeader({ showMetrics: false, showActions: true });
+const headerWithMetrics = (props, t) =>
+  projectTableHeader({ showMetrics: true, showActions: true }, t);
```

Comment:
It looks like `projectTableHeader` no longer takes a `t` parameter?

Reply:
@spadgett , yes you are correct!